lib/src/screens/backup/backup_page.dart

```diff
@@ -52,8 +52,7 @@ class BackupPage extends BasePage {
                       child: Observer(
                           builder: (_) => GestureDetector(
                                 onTap: () {
-                                  Clipboard.setData(
-                                      ClipboardData(text: backupViewModelBase.backupPassword));
+                                  SensitiveClipboard.copy(backupViewModelBase.backupPassword);
```

Comment:
check here if the platform is mobile then use this package if not, then use the old method